### PR TITLE
Karma menu update

### PR DESCRIPTION
**Pull request by ZomgPonies**
## What Does This PR Do
Adds current karma amount to karma menu, and also updates karma menu to show already unlocked jobs and species. This does NOT affect awarding karma, it is read-only usability changes.

## Why It's Good For The Game
The karma menu was implemented in 2013 and had not been touched up much since. It is due to be brought more in line with the usability of other UI menus.

## Images of changes
Capture

## Changelog
:cl:
fix: karma menu not showing current karma
fix: karma menu not showing unlocked jobs/species
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Could we get an image of what this looks like?

File: code/modules/karma/karma.dm [hunk 1]
```diff
@@ -195,6 +182,23 @@ GLOBAL_LIST_EMPTY(karma_spenders)
 	karmashopmenu()
 
 /client/proc/karmashopmenu()
+	var/DBQuery/query = GLOB.dbcon.NewQuery("SELECT * FROM [format_table_name("whitelist")] WHERE ckey='[usr.ckey]'")
```
Comment (AffectedArc07) [inline code/modules/karma/karma.dm:185]: Sanitize the ckey. Yes I know its likely someone will name their account `Robert'); DROP TABLE Students;--` but you can never be so sure

Comment (Kyep) [reply code/modules/karma/karma.dm:185]: Ah yes, little bobby tables, we call him.

Comment (ZomgPonies) [reply code/modules/karma/karma.dm:185]: xkcd never gets old.

**Comment by Spacemanspark** [comment]
I'd rather we had the verb there than not honestly. Rest of the changes are pretty nice, however.

File: code/modules/karma/karma.dm (hunk 1 above)
Comment (ZomgPonies) [reply code/modules/karma/karma.dm]: Sanitation added all round

**Comment by AffectedArc07** [comment]
image
Move this up to the top as a header before the purchases (here for example)
image
Also line break the end of it to give a distinction between the two

**Comment by AffectedArc07** [comment]
Perfect

**Comment by KasparoVy** [comment]
Only one ya need is Vox mate

**Comment by Spacemanspark** [comment]
Kasparov has been banned

**Comment by Fox-McCloud** [comment]
>Only one ya need is Vox mate

The only karma unlock needed is no karma unlocks.

**Pull request merged**